CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased
 - PHP 8 support
 - Change the error handling for silenced fatal errors using `@` to use a mask check in order to be php 8 compatible (#1141)
+- Change the dependency of `guzzlehttp/promises` from `^1.3` to `^1.4` since it contains crucial type checks for the `FulfilledPromise` class (#1144)
```

Comment:
We can shorten this phrase to ``Update the `guzzlehttp/promises` package to the minimum required version compatible with PHP 8``, WDYT?

Reply:
Sure, I guess if one want's to find out more about the reason he still has the PR reference

Reply:
changed